packages/lesswrong/components/posts/AllPostsPage.tsx

```diff
@@ -94,7 +94,7 @@ const AllPostsPage = ({classes}: {classes: ClassesType}) => {
               <SectionTitle title={preferredHeadingCase("All Posts")}>
                 {isFriendlyUI ?
                   <SortButton label={formatSort(currentSorting)} /> :
-                  <SettingsButton color="black" label={`Sorted by ${ SORT_ORDER_OPTIONS[currentSorting].label }`}/>
+                  <SettingsButton textShadow label={`Sorted by ${ SORT_ORDER_OPTIONS[currentSorting].label }`}/>
```

Comment:
My guess is that this particular instance should be gated on `!isFriendlyUI`; I'm not sure what this does on the EA forum all posts page.